WebFileSystem.ts

```diff
@@ -613,12 +613,12 @@ class WebFileSystem extends webdav.FileSystem {
                 if (await this.loadPath(path, user)) {
                     callback(null, this.resources.get(user.uid).get(path.toString()).type)
                 } else {
-                    logger.error('File could not be found!')
+                    logger.error(`WebFileSystem._type : File could not be found! uid: ${info.context.user.uid} path: ${path.toString()}`, new Error('Stack-Tracer'))
```

Comment:
This error could be a bit more accurate with the condition returning `false`, so we only know, it happened in the `_type`-method

Reply:
This should be mentioned in the Error tracer, i believe, i also added it in the message though